website/content/docs/job-specification/volume.mdx

```diff
@@ -76,6 +76,11 @@ the [volume_mount][volume_mount] block in the `task` configuration.
   used for validating `host_volume` ACLs and for scheduling when a
   matching `host_volume` requires `read_only` usage.
 
+- `sticky` `(bool: false)` - Specifies that this volume is supposed to "stick"
+  to the allocation that uses it. This means that upon every reschedule and
+  replacement, the task group will always receive the volume with the same ID, if
+  available.
```

Comment:
We should note here this can only be used with dynamic host volumes. This would also be a good place to cross-link to task volume claims and stateful deployments page.